lambda-http4s/src/main/scala/feral/lambda/http4s/ApiGatewayProxyHandler.scala

```diff
@@ -39,12 +43,21 @@ object ApiGatewayProxyHandler {
     for {
       event <- LambdaEnv.event
       request <- decodeEvent(event)
-      response <- routes(request).getOrElse(Response.notFound[F])
+      response <- routes(request).getOrElse(Response.notFound)
       isBase64Encoded = !response.charset.contains(Charset.`UTF-8`)
-      responseBody <- (if (isBase64Encoded)
-                         response.body.through(fs2.text.base64.encode)
-                       else
-                         response.body.through(fs2.text.utf8.decode)).compile.foldMonoid
+      responseBody <- response.entity match {
+        case Entity.Empty => "".pure
+        case Entity.Strict(chunk) => // TODO
+          if (isBase64Encoded) chunk.toByteVector.toBase64.pure
+          else chunk.toByteVector.decodeUtf8.liftTo
```

Comment:
This makes me grumpy. I still think `Entity.Strict` should use a `ByteVector` instead of a `Chunk`.